ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -967,6 +967,45 @@ def execute_expiring_check(check):
             # synced. Help chrony by resetting the date
             self.master.run_command(['date', '-s', now_str])
 
+    @pytest.fixture
+    def rename_ldif(self):
+        """Fixture to rename dse.ldif file and revert after test"""
+        instance = realm_to_serverid(self.master.domain.realm)
+        self.master.run_command(
+            [
+                "mv",
+                paths.ETC_DIRSRV_SLAPD_INSTANCE_TEMPLATE % instance
+                + "/dse.ldif",
+                paths.ETC_DIRSRV_SLAPD_INSTANCE_TEMPLATE % instance
+                + "/_dse.ldif",
```

Comment:
The _dse seems a bit subtle. Can you name it something like dse.ldif.rename or something? I had to stare at this a bit to see the single-character difference.

Reply:
@rcritten, the file has been changed to dse.ldif.renamed.